firebase-vscode/webpack.common.js

```diff
@@ -309,4 +309,9 @@ module.exports = [
     .filter((filename) => filename.match(/\.entry\.tsx/))
     .map((filename) => filename.replace(/\.entry\.tsx/, ""))
     .map((name) => makeWebConfig(name)),
+  ...fs
```

Comment:
Nit - this is a lot of logic to inline. For clarity, consider making these 2 long chained calls into variables ie:
```
const webConfigs =   ...fs
    .readdirSync("webviews")
    .filter((filename) => filename.match(/\.entry\.tsx/))
    .map((filename) => filename.replace(/\.entry\.tsx/, ""))
    .map((name) => makeWebConfig(name));
const dataconnectWebConfigs = <blah>

module.exports = [ extensionConfig, ...webConfigs, ...dataconnectWebConfigs ];

Reply:
done